### ISSUE TITLE
Tab bar buttons should have a minimum size when adding lots of tabs

### ISSUE THREAD

**Issue by Altonss** [skyjake/lagrange#513]
n tabs:
![image](https://user-images.githubusercontent.com/66519591/171058016-ee113839-80a1-4e9e-827e-a3be9e0424f2.png)
n+1 tabs:
![image](https://user-images.githubusercontent.com/66519591/171058056-5caca52e-aab9-4097-837c-b06c3c5e7ecd.png)
n+2 tabs:
![image](https://user-images.githubusercontent.com/66519591/171058100-f38da3da-909f-441f-ba4a-7e0b8423ff06.png)

**Comment by skyjake** [comment]
Yeah, the tab bar needs more work... In addition to handling a large number of tabs, it also needs drag-and-drop for reordering them.
